modules/wallet/money.go

```diff
@@ -111,6 +113,62 @@ func (w *Wallet) SendSiacoins(amount types.Currency, dest types.UnlockHash) ([]t
 	return txnSet, nil
 }
 
+// SendSiacoinsMulti creates a transaction sending each amount in 'amounts' to
+// it's corresponding destination in 'dests'. The transaction is submitted to
+// the transaction pool and is also returned.
+func (w *Wallet) SendSiacoinsMulti(amounts []types.Currency, dests []types.UnlockHash) ([]types.Transaction, error) {
+	if err := w.tg.Add(); err != nil {
+		return nil, err
+	}
+	defer w.tg.Done()
+	if !w.unlocked {
+		w.log.Println("Attempt to send coins has failed - wallet is locked")
+		return nil, modules.ErrLockedWallet
+	}
+	if len(amounts) != len(dests) {
+		return nil, errors.New("number of amounts and destinations must match")
+	}
+
+	txnBuilder := w.StartTransaction()
+
+	// Add estimated transaction fee.
+	_, tpoolFee := w.tpool.FeeEstimation()
+	tpoolFee = tpoolFee.Mul64(750 * uint64(len(dests))) // Estimated transaction size in bytes
```

Comment:
can no longer use 750 as a reasonable guess. Should make some trial transactions to see. A send-to-5, send-to-50, etc.

I'm guessing it'll take somewhere around 150 bytes per destination+amount, but  really we should measure that.